### PR TITLE
Fix golint for loadBalanceSegmentLoop

### PR DESCRIPTION
**Pull request by xige-16**
/kind improvement
Signed-off-by: xige-16 <xi.ge@zilliz.com>

### CODE REVIEW

File: internal/querycoord/query_coord.go [hunk 1]
```diff
@@ -434,7 +434,7 @@ func (qc *QueryCoord) loadBalanceSegmentLoop() {
 	ctx, cancel := context.WithCancel(qc.loopCtx)
 	defer cancel()
 	defer qc.loopWg.Done()
-	log.Debug("query coordinator start load balance segment loop")
+	log.Debug("QUeryCoord start load balance segment loop")
```
Comment (congqixia) [inline internal/querycoord/query_coord.go:437]: QUeryCoord => QueryCoord

Comment (xige-16) [reply internal/querycoord/query_coord.go]: fixed

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**